docs/nda.md

```diff
@@ -0,0 +1,70 @@
+NDA Policy
+==========
+
+We, as maintainers of the fwupd and LVFS projects often get asked by companies to sign NDAs and
+we’ve always said no.
+Rather than rewriting a version of this document for each individual request, we’ve summarized the
+key points here.
+
+Before we sign an NDA, we need to spend time to check it, and have a lawyer check it.
+The protection language of an NDA is almost always too broad, vague and ambiguous because usually,
+it’s a “generic NDA” and we need to negotiate to make it more specific, with our lawyer and the
+companies lawyer involved.
+
+That means money and time – which is a really bitter pill to swallow if the company isn’t actually
+contributing financially to fwupd/LVFS.
+The remediation language is almost always open-ended and requires that we agree to eventualities
+that are nearly impossible to define fully.
+We often see language like “irreparable harm” and “cannot adequately be compensated with damages,” etc.
+This means the potential negative outcomes are near limitless
```

Comment:
This sentence is not finished with a dot

Reply:
Good catch, fixed, thanks.